lib/ClimaCoreTempestRemap/Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ClimaCoreTempestRemap"
 uuid = "d934ef94-cdd4-4710-83d6-720549644b70"
 authors = ["CliMA Contributors <clima-software@caltech.edu>"]
-version = "0.3.7"
+version = "0.3.8"
 
```

Comment:
We never actually tagged 0.3.7

Reply:
So you don't need to update this

Reply:
I seems to be tagged here: https://github.com/CliMA/ClimaCore.jl/commit/e5a2ec9c89f835198c25e44fe8c578c2f439b427#diff-1fa6758b8a92bc430e50639e6151ba4f28598df076aee726a46f8c821e86acce
Not sure why this wasn't in the registry!

Reply:
looks like registrator didn't pick it up

Just up the min version, then we can tag it